4-nodarbiba/uzdevumi/DenissMijusovs/objects/js/app.js

```diff
@@ -32,5 +32,33 @@
 // 7. Izvadīt konsolē CV objekta īpašības workExperience 2 elementu (index 1).
 // 8. Izvadīt konsolē CV objekta īpašības languages objekta eng īpašības name un level apvienojumu.
 //    -- piem., vardsUzvards.languages.lv objekta īpašību apvienojums izskatītos šādi:
-//       Latviešu - Native 
-//       Un tiktu panākta ar string literal pierakstu `${vardsUzvards.languages.lv.name} - ${vardsUzvards.languages.lv.level}`
\ No newline at end of file
+//       Latviešu - Native
+//       Un tiktu panākta ar string literal pierakstu `${vardsUzvards.languages.lv.name} - ${vardsUzvards.languages.lv.level}`
+
+const denissMijusovs = {
+  firstName: "Deniss",
+  lastName: "Mijusovs",
+  email: "dmij*****@gmail.com",
+  telNr: 23000000,
+  workExperience: ["McDonalds", "Atea", "Izoterms", "ON24"],
+  languages: {
+    lv: {
+      name: "Latviešu",
+      level: "Izcili",
+    },
+    rus: {
+      name: "Krievu",
+      level: "Native",
+    },
+    eng: {
+      name: "Angļu",
+      level: "Ļoti labi",
+    },
+  },
+};
+
+console.log(denissMijusovs.firstName + " " + denissMijusovs.lastName);
```

Comment:
Šis, protams, strādā un ir ok.
Bet labi būtu izmantot "string literals" sintaksi:

`'${denissMijusovs.firstName} ${denissMijusovs.lastName}'`